hack/tests/subcommand-scorecard.sh

```diff
@@ -44,7 +44,7 @@ header_text 'scorecard test to see if bundle flag works correctly'
 if ! commandoutput="$(operator-sdk scorecard --config "$CONFIG_PATH_BUNDLE" 2>&1)"; then 
 	echo $commandoutput
 	failCount=`echo $commandoutput | grep -o "fail" | wc -l`
-	expectedFailCount=2
+	expectedFailCount=3
```

Comment:
Curious why this changed?

Reply:
I had found a test that I think would never return a failure, I fixed that test which changed the fail count in the test.  The test was in the OLM suite, CRDsHaveResourcesTest.  In this set of commits I've added logic in there to set pass/fail.